shared/containers/Community/index.js

```diff
@@ -1,6 +1,14 @@
 import { connect } from 'react-redux';
 import Community from '../../components/Community';
 
+function featuredEvent(events) {
+  return events.find(e => e.displayLevel === 'Featured') || {};
+}
+
+function futureEvents(events) {
+  return events.filter(e => e.displayLevel === 'Highlighted').slice(0, 3);
```

Comment:
I know I mentioned this before - but the args to .slice - would be nice if they were constants. e.g. FIRST_EVENT_POS, LAST_EVENT_POS